src/core/test/fuzzing/src/test/scala/FuzzingTest.scala

```diff
@@ -26,6 +26,7 @@ class FuzzingTest extends TestBase {
     val exemptions: Set[String] = Set(
       "com.microsoft.ml.spark.TrainClassifier",
       "com.microsoft.ml.spark.TrainRegressor",
+      "com.microsoft.ml.spark.TuneHyperparameters",
```

Comment:
Can we try to get this passing the fuzzing tests? It will be easier if you dont have the directory running issue

Reply:
sure, the directory issue is independent of the fuzzing tests

Reply:
Can you remove this line?

Reply:
Still here

Reply:
(as discussed I re-added this back for now due to a different issue in fuzzing)